poly-commitment/src/evaluation_proof.rs

```diff
@@ -130,9 +130,11 @@ impl<G: CommitmentCurve> SRS<G> {
                     scale *= &polyscale;
                     offset += self.g.len();
                     if let Some(m) = degree_bound {
-                        if offset > *m {
-                            // mixing in the shifted segment since degree is bounded
-                            plnm.add_shifted(scale, self.g.len() - m % self.g.len(), segment);
+                        if offset >= *m {
```

Comment:
perhaps it'd be a good idea to start avoiding "degree" and use `size` or `length`

cf https://twitter.com/cronokirby/status/1568268593946677248